packages/@lwc/engine-core/src/framework/rendering.ts

```diff
@@ -246,6 +255,28 @@ function mountStatic(
     insertNode(elm, parent, anchor, renderer);
 }
 
+function mountFragment(
+    vnode: VFragment,
+    parent: ParentNode,
+    anchor: Node | null,
+    renderer: RendererAPI
+) {
+    const fragmentStart = (vnode.elm = renderer.createText(''));
+    const fragmentEnd = (vnode.anchor = renderer.createText(''));
```

Comment:
These text nodes seem to be causing failures in downstream Nucleus tests due to snapshots having extra unexpected whitespace.

Not sure there's anything we can do about it, but just pointing it out.